samples/buildstrategy/buildah/buildstrategy_buildah_shipwright_managed_push_cr.yaml

```diff
@@ -193,6 +193,11 @@ spec:
       defaults:
         - docker.io
         - quay.io
+    # - name: storage-driver
+    #   description: "The storage driver to use, such as `overlay` or `vfs`."
+    #   type: string
+    #   default: "vfs"
+      # For details see the "--storage-driver" section of https://github.com/containers/buildah/blob/main/docs/buildah.1.md#options
```

Comment:
Did you mean to leave this as commented out?

Reply:
Yes. I was just trying some checks. I'll push the suggested changes soon.